### PR TITLE
Replaced the base image jboss/base-jdk:8 with UBI8 in the lra-coordinator

### PR DESCRIPTION
**Pull request by jmfinelli**
This PR is to update the base image used in all `Dockerfile`s in this repository.

### CODE REVIEW

File: lra/lra-coordinator/README.adoc [hunk 1]
```diff
@@ -1,75 +1,56 @@
...
-  The permitted values are log levels defined by Quarkus runtime (https://quarkus.io/guides/logging).
-  The default log level value is `INFO`.
-`OBJECT_STORE_PATH`::
-  It could be defined on docker run.
-  It defines a directory where the transaction log is stored. Transaction log is the persistent storage
-  of the LRA coordinator. The default location is directory `/data`.
+* `NARAYANA_VERSION` is used only during the `build` phase and defines what version of Narayana's LRA coordinator  will be downloaded
+* `NARAYANA_FULL_URL` is used during the `build` phase to set up the location where the Narayana distro will be downloaded from
+* `LOG_LEVEL` is used during the `run` phase to set up the level/accuracy of logging produced by the LRA coordinator. The permitted values are logging levels defined in the [Quarkus runtime](https://quarkus.io/guides/logging). The default logging level value is `INFO`.
+* `OBJECT_STORE_PATH` is used during the `run` phase to set up a directory where the transaction log is stored (which is the persistent storage of the LRA coordinator). The default location is the directory `/data`.
```
Comment (mmusgrov) [inline lra/lra-coordinator/README.adoc:12]: Thanks for explicitly stating that the storage needs to be persistent.

File: lra/lra-coordinator/README.adoc [hunk 2]
```diff
@@ -1,75 +1,56 @@
...
 ```bash
-docker pull docker.io/jbosstm/lra-coordinator
+podman pull quay.io/jbosstm/lra-coordinator:latest
 ```
 
-== Release the lra docker image to Dockerhub
+== Release the lra docker image
 
-If you build the image you can push the new version to dockerhub with sequence
+After building the image of `lra-coordinator`, it is possible to tag and push the new version to `quay.io` following the commands:
```
Comment (mmusgrov) [inline lra/lra-coordinator/README.adoc]: Add a comment showing how to build the image:
  `podman build --tag lra-coordinator --build-arg NARAYANA_VERSION=${NARAYANA_VERSION} .`

**Review by mmusgrov — APPROVED** [review]
I had just the one comment.

File: lra/lra-coordinator/README.adoc (hunk 2 above)
Comment (jmfinelli) [reply lra/lra-coordinator/README.adoc]: Added a reference to the `Build` section.

**Comment by jmfinelli** [comment]
Thanks for the review @mmusgrov. Could you please merge it as well?

**Pull request merged**